objects_spawns: {a: [ [-2.5, 1.25, 1.25], [-1, -7, 1.02] ], 
                 b: [ [-3.5, -10.5, 0.81], [-1.6, -4.5, 0.52]],
                 c: [ [-3.42, -7.16, 0.81], [2.42, -12.07, 1.20], [2.86, -8.29, 0.81] ]}
